elementary/clients/dbt/dbt_runner.py

```diff
@@ -156,3 +160,15 @@ def test(
             command_args=command_args, vars=vars, quiet=quiet
         )
         return success
+
+    def _get_command_env(self):
+        env = os.environ.copy()
+        if self.dbt_quoting is not None:
```

Comment:
I really like the option to pass env vars to the dbt runner!
IMO we should make it more generic and instead of passing a specific key that we need to manually map to the envs (and create a new cli variable for it), we should do something like that:
```
if self.dbt_env_vars is not None:
    env.update({**self.dbt_env_vars})
```

That way we can easily make new env vars supported in the future without changing a thing!
WDYT?

Reply:
It's a cool idea!
My only thought is that in this case I'm passing internal env variables that I made up (e.g. not something that is directly supported by dbt)
So for this particular use case maybe an explicit CLI parameter is good? WDYT?

But nonetheless maybe I can change the code to support both.